apps/prairielearn/src/tests/sync/tagsTopicsSync.test.ts

```diff
@@ -23,10 +22,10 @@ function makeEntity() {
  * Checks that the entity present in the database matches the data
  * from the original entity in `infoCourse.json`.
  *
- * @param {any} syncedEntity - The entity from the database.
- * @param {any} entity - The entity from `infoCourse.json`.
+ * @param syncedEntity - The entity from the database.
+ * @param entity - The entity from `infoCourse.json`.
```

Comment:
Can we remove the JSDoc comments here? There were some others throughout the PR as well but just leaving the comment here in case there is a reason we are keeping them.

Reply:
JSDoc is still useful. IDEs like vscode will provide information about a function based on these when you hover over the name in other files.

Reply:
<img width="568" alt="image" src="https://github.com/PrairieLearn/PrairieLearn/assets/1004907/5db608db-6e50-47b9-9dec-4404de4b85fb">

Reply:
In case this helps describe it: there's a subtle difference between JSDoc comments that describe the purpose/params/return-value of function, and those that describe the *types* of the parameters and return values. I converted the types in such comments to native types (since `tsc` won't pick up JSDoc types in `.ts` files), but I left the written descriptions intact.